CONTRIBUTING.md

```diff
@@ -34,15 +34,15 @@ A pull request will be considered if:
 * You can respond to feedback quickly.
 * It touches fewer than about 10 files and the changes are less than about 200 lines. Changes larger than this may be OK, or you may be asked to submit multiple pull requests of a more manageable size.
 * Each logical change is a separate commit within the pull request. Commit messages for each change should follow the [commit log message guide](https://docs.freebsd.org/en/articles/committers-guide/#commit-log-message).
-* All commits have your name and valid email address as you would like to see them in the FreeBSD repository as the author. Fake github.com addresses cannot be used.
+* All commits have, as the author, your name and valid email address as you would like to see them in the FreeBSD repository. Fake github.com addresses cannot be used.
 * The scope of the pull request should not change during review. If the review suggests changes that expand the scope, please create an independent pull request.
 * Fixup commits should be squashed with the commit they are fixing. Each commit in your branch should be suitable for FreeBSD's repository.
 * Commits should include one or more `Signed-off-by:` lines with full name and email address certifying [Developer Certificate of Origin](https://developercertificate.org/).
 * The commits follow FreeBSD's style guide. See [Style](#Style).
-* Run tools/build/checkstyle9.pl on your git branch and eliminate all errors
-* The commits should not introduce trailing white space.
+* Run tools/build/checkstyle9.pl on your Git branch and eliminate all errors
+* The commits do not introduce trailing white space.
 * If the commmit fixes a bug, please add 'PR: <bugnumber>' to the comment message.
```

Comment:
Grammatically, list items such as this are not a perfect fit with the sentence that precedes the list, e.g.: 

> A pull request will be considered if: If the commmit fixes a bug, please add 'PR: \<bugnumber>\' to the comment message.

– but let's not slave over it.

Reply:
Yea, I'm open to better introductory wording...  but I'm going to land this pull-request as-is.